plan_and_progress/2024_11.md

```diff
@@ -10,12 +10,14 @@
 - 27 (Wed): 
     - Recall this project
     - Try chunking
-    - Write some NLP basics description
+    - Write some NLP basics description)
```

Comment:
`nit` you might have missed to add the left parenthesis

Reply:
Thank you! I'm trying to fix it now. Im learning to use GitHub from iPhone app so that I can work on this more!